openmmforcefields/generators/template_generators.py

```diff
@@ -878,7 +905,8 @@ def read_file_contents(filename):
 
             # Run parmchk.
             shutil.copy(self.gaff_dat_filename, "gaff.dat")
-            cmd = f"parmchk2 -i out.mol2 -f mol2 -p gaff.dat -o out.frcmod -s {self._gaff_major_version}"
+            cmd = f"parmchk2 -i out.mol2 -f mol2 -p gaff.dat -o out.frcmod -s {self._gaff_major_version} -a Y"
```

Comment:
This is the most important part, here we are telling `parmchk2` to print out all of the parameters. We used to print out only parms that didn't exist in the gaff.dat file. What this enables:

1) we don't have to convert new gaff.dat versions to xmls
2) we can support whatever gaff versions that `parmchk2` supports (from the given ambertools version)